content/en/docs/other-guides/kubeflow-behind-proxy.md

```diff
@@ -67,3 +67,48 @@ kfctl build -V -f kfctl_k8s_istio.yaml
 ```shell
 kfctl apply -V -f kfctl_k8s_istio.yaml
 ```
+
+# To Use a ConfigMap to Configure Kubeflow Proxy
+An alternative to changing the manifests would be to use `PodDefault` to inject the proxy configuration via a (ConfigMap)[https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/] 
```

Comment:
Suggestion:

- Fix the link: Markdown's format is `[ a thing ](link)` instead of `(a thing)[link]`
- Use sentence case

```suggestion

# To use a ConfigMap to configure the Kubeflow proxy

An alternative to changing the manifests would be to use `PodDefault` to inject the proxy configuration via a [ConfigMap](https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/).

```